app/src/main/java/com/example/sharewhatyoucanproject/LoginActivity.kt

```diff
@@ -62,170 +56,155 @@ class LoginActivity : AppCompatActivity() {
         namearray.add("William")
         namearray.add("Henry")
         namearray.add("Benjamin")
-
-
         submitdevice = findViewById(R.id.submitdevice)
-
-        submitdevice.setOnClickListener(View.OnClickListener {
-            deviceid = Settings.Secure.getString(contentResolver, Settings.Secure.ANDROID_ID)
-            email = "s" + deviceid + "@gmail.com"
-
-
-
-            pd.show()
-            checkuser()
-
-        })
-
-
+        submitdevice.setOnClickListener(
+            View.OnClickListener {
+                deviceid = Settings.Secure.getString(contentResolver, Settings.Secure.ANDROID_ID)
+                email = "s" + deviceid + "@gmail.com"
+                pd.show()
+                checkuser()
```

Comment:
Use camel case.  Also, what do you mean by "check user"?  Is there a better name for this function?